packages/fxa-content-server/app/scripts/templates/pair/index.mustache

```diff
@@ -1,37 +1,89 @@
-<div class="card">
-  <header>
-    {{#showQrCode}}
-      <h2 id="pair-header" class="card-header">{{#t}}Connect Firefox on another device{{/t}}</h2>
-    {{/showQrCode}}
-    {{^showQrCode}}
+
+<div class="card-base">
+  <header class="relative">
+    {{#askMobileStatus}}
+      {{#needsMobileConfirmed}}
+        <button id="back-btn" class="me-4 tablet:me-0 tablet:p-4 inline-block tablet:absolute tablet:-start-20 -top-1.5 w-5 h-3.5 p-5 bg-back-arrow bg-auto bg-center bg-no-repeat rtl:transform rtl:-scale-x-100" aria-label="{{#t}}Back{{/t}}" title="{{#t}}Back{{/t}}"></button>
+      {{/needsMobileConfirmed}}
+      <h1 class="mb-5 text-base{{#needsMobileConfirmed}} inline-block align-top mt-2 tablet:mt-0{{/needsMobileConfirmed}}">Connect another device</h1>
+
+      {{^needsMobileConfirmed}}
+        <h2 id="pair-header" class="card-header">{{#t}}Sync your Firefox experience{{/t}}</h2>
+      {{/needsMobileConfirmed}}
+      {{#needsMobileConfirmed}}
+        <h2 id="pair-header-mobile" class="card-header">{{#t}}Download Firefox for mobile{{/t}}</h2>
+      {{/needsMobileConfirmed}}
+
+    {{/askMobileStatus}}
+    {{^askMobileStatus}}
       <h2 id="pair-header" class="card-header">{{#t}}Sync Firefox on your phone or tablet{{/t}}</h2>
-    {{/showQrCode}}
+    {{/askMobileStatus}}
   </header>
 
   <section>
     <div class="error"></div>
 
-    {{#showQrCode}}
-    <div class="mt-9 mb-5">{{#t}}Already have Firefox on a phone or tablet?{{/t}}</div>
-    <form novalidate>
-      <div class="flex">
-        <button id="start-pairing" class="cta-primary cta-xl" type="submit">{{#t}}Sync your device{{/t}}</button>
-      </div>
-    </form>
-
-      <h3 class="text-lg font-bold mt-10">{{#t}}Or download{{/t}}</h3>
-      <p class="my-5 px-16">{{#unsafeTranslate}}Scan to download Firefox for mobile, or send yourself a <a id="get-fx-mobile" href="https://www.mozilla.org/firefox/mobile/get-app/" target="_blank" class="link-blue">download link.</a>{{/unsafeTranslate}}</p>
-      <div class="bg-image-cad-qr-code" role="img" aria-label="{{#t}}QR code{{/t}}"></div>
-    <p class="mt-5 text-sm"><a id="pair-not-now" class="link-blue" href="/settings">{{#t}}Not now{{/t}}</a></p>
-    {{/showQrCode}}
-    {{^showQrCode}}
-    <div class="{{graphicId}} mt-5" role="img" aria-label="{{#t}}Complete your set-up{{/t}}"></div>
-    <p class="mt-5 mb-5 px-16">{{#t}}Take your tabs, bookmarks, and passwords anywhere you use Firefox.{{/t}}</p>
-    <form novalidate>
-      <div class="flex">
-        <button id="start-pairing" class="cta-primary cta-xl" type="submit">{{#t}}Get started{{/t}}</button>
-      </div>
-    </form>
-    {{/showQrCode}}
+    {{#askMobileStatus}}
+      {{^needsMobileConfirmed}}
+        <p class="my-3 text-base">{{#t}}View your saved passwords, tabs, browsing history and more — across all your devices.{{/t}}</p>
+
+        <form novalidate>
+          <fieldset>
```

Comment:
Nit: to my eye there is a bit too much spacing between the radio options and the submit button. Perhaps `my-5` on fieldset and no margin on button?
![image](https://github.com/mozilla/fxa/assets/22231637/64322ed8-7bfb-45cb-83c8-824858be76aa)

Reply:
@vpomerleau Let me push another update, I just made a couple of tweaks here.